### PR TITLE
Add visit:region and visit:city property entries

### PR DESCRIPTION
**Pull request by 68kHeart**
I've added the `visit:region` property to the Stats API docs. I need information about what standard or data set the `visit:city` codes are from and I'll be happy to write a commit for that, too.

An example mapping is `4347778` to Baltimore, Maryland, USA.

### CODE REVIEW

**Comment by 68kHeart** [comment]
Thanks, PR is good for review now!

File: docs/stats-api.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ more depth. Here's the full list of properties we collect automatically:
 | visit:os                   | Mac                           | Name of the operating system. Most popular ones are `Mac`, `Windows`, `iOS` and `Android`. Linux distributions are reported separately |
 | visit:os_version           | 10.6                          | Version number of the operating system used by the visitor                                                                            |
 | visit:country              | US                            | ISO 3166-1 alpha-2 code of the visitor country                                                                                        |
-
+| visit:region               | US-MD                         | ISO 3166-2 code of the visitor region                                                                                                  |
+| visit:city                 | 4347778                       | [GeoName ID](http://download.geonames.org/export) of the visitor region                                                                |
```
Comment (ukutaht) [inline docs/stats-api.md:87]: city* not region

**Comment by ukutaht** [comment]
No worries about the commits

**Comment by metmarkosaric** [comment]
thanks @68kHeart!

**Comment by i8ramin** [comment]
Hello. How does one map a geoname id to an actual human readable city name?

**Comment by ruslandoga** [comment]
👋 @i8ramin 

You can try using http://www.geonames.org. For example, searching for london returns http://www.geonames.org/2643743/london.html as the first entry, and its geoname id is 2643743.

**Pull request merged**